converters/pokemon.py

```diff
@@ -288,20 +298,56 @@ def convert_pdata(input_csv, header=DEFAULT_HEADER):
         evolve = Evolve(header, pokemon_list)
         filter_data = FilterData(header)
         index_order = IndexOrder(header)
+        
+        poke_by_name = {}
 
+        # Collect all the rows into Pokemon types
         for index, row in enumerate(reader, 1):
             if not row:
                 continue
 
             # Each row is one Pokemon
             poke = Pokemon(header)
             poke.setup(row)
+            poke_by_name[poke.name] = poke
+
+        # Some rows are variants of a single pokemon type. Let's go collect those
+        variant_by_type = {}
+        def_poke_by_type = {}
+        def_type_by_variant = {}
+        # TODO: In the future we may have other variants, like Alolan forms or something.
+        # It's unclear what those might look like from a data perspective
+        for name,variants in util.VARIANT_DATA.items():
+            if type(variants) is list:
+                for d in variants:
```

Comment:
I want bananas with the "naming" comments so might as well make this into a `data` or something more descriptive.